tests/unit/acl.tcl

```diff
@@ -7,6 +7,10 @@ start_server {tags {"acl external:skip"}} {
         r ACL setuser newuser
     }
 
+    test {Basic ACL USERS} {
+        r ACL USERS
```

Comment:
was this uncovered?
same question for ASKING and LASTSAVE?

Reply:
i see we have quite a few new tests for coverage, and i guess they're just for various replies not that the command was completely uncovered.
maybe we don't have to list each of these in the top comment, but let's at least make sure there's a bullet that mentions we improved test coverage and explain roughly what we did and how